### PR TITLE
Swift Package Manager support

### PR DESCRIPTION
**Pull request by domasn**
Screenshot 2020-04-22 at 11 47 14

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.0
```
Comment (eimantas) [inline Package.swift:1]: Probably only this comment is needed 🧹

**Review by eimantas — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (domasn) [reply Package.swift:1]: Nice! I just did the default project init.

Comment (domasn) [reply Package.swift:1]: Doesn't work if I only leave one line in this file 😞 

Screenshot 2020-04-22 at 13 10 04

Comment (eimantas) [reply Package.swift:1]: One line as in only that comment with rest of package description :D

**Pull request merged**